pkgs/applications/editors/vim/plugins/overrides.nix

```diff
@@ -847,6 +847,10 @@ self: super: {
     dependencies = with self; [ plenary-nvim ];
   };
 
+  neotest = super.neorg.overrideAttrs {
+    dependencies = with self; [ FixCursorHold-nvim plenary-nvim ];
```

Comment:
Given that the README says it shouldn't be needed after https://github.com/neovim/neovim/pull/20198, I'd remove it.

Reply:
Indeed !
Done.